frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleash.tsx

```diff
@@ -134,6 +146,35 @@ export const NewInUnleash = ({
                 </>
             ),
         },
+        {
+            label: 'Event timeline',
+            summary: 'Keep track of recent events across all your projects',
+            icon: <StyledLinearScaleIcon />,
+            preview: <EventTimelinePreview />,
+            onCheckItOut: () => {
+                showTimeline(true, { highlight: true });
+                window.scrollTo({
+                    top: 0,
+                    behavior: 'smooth',
+                });
+            },
+            docsLink: 'https://docs.getunleash.io/reference/events',
```

Comment:
@melindafekete we may want to update this to a more event-timeline-specific documentation URL once it’s available.